src/main/java/org/broadinstitute/consent/http/db/DataSetDAO.java

```diff
@@ -196,7 +196,7 @@
     @SqlQuery("SELECT dataSetId FROM dataset WHERE name = :name")
     Integer getDatasetIdByName(@Bind("name") String name);
 
-    @SqlQuery("SELECT * FROM dataset WHERE LOWER(name) LIKE CONCAT('%', LOWER(:name), '%')")
+    @SqlQuery("SELECT * FROM dataset WHERE LOWER(name) = LOWER(:name);")
```

Comment:
We don't need the semi-colon, but I don't think it hurts either. Best to remove it for consistency.